src/AvatarStack.js

```diff
@@ -0,0 +1,159 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import {get, COMMON} from './constants'
+import theme from './theme'
+
+const rightStyles = `
+  .body {
+    right: 0;
+    flex-direction: row-reverse;
+
+    &:hover .avatar {
+      margin-right: 0;
+      margin-left: 3px;
+    }
+  }
+
+  .avatar.avatar-more {
+    background: ${get('colors.gray.3')};
+
+    &::before {
+      width: 5px;
+    }
+
+    &::after {
+      width: 2px;
+      background: ${get('colors.gray.1')};
+    }
+  }
+
+  .avatar {
+    margin-right: 0;
+    margin-left: -11px;
+    border-right: 0;
+    border-left: ${get('borders.1')} solid ${get('colors.white')};
+  }
+`
+
+const Item = styled.img.attrs(() => ({
+  className: 'avatar'
```

Comment:
Is there a consistent naming convention we want to use for these internal classNames?  `<UnderlineNav />` uses this `.UnderlineNav-body` whereas the `<FilterList.Item />` uses this: `.count`.

I don't have a strong opinion either way - unless we want something like COMPONENT-SUBCOMPONENT--OPTION, then I say we just keep it simple with lowercase and - like you have in this PR, and maybe file an issue to keep up `<UnderlineNav />`

Reply:
No, we don't really have a naming convention for their internal class names! They should probably be more specific though so they component doesn't inherit other CSS that might be in the page 🤔 I will update this classname!